template/README.md

```diff
@@ -2,19 +2,124 @@
 
 A brief description of My Project.
 
+## Using This Template
+
+Remove this section after initial setup!
+
+Search for and replace the following placeholders:
+
+* \<organization>
+* \<project>
+* \<project local url>
+* \<local environment>
+* \<project key>
+* \<branch>
+* \<provide link>
+
 ## Getting Started
 
-This project is based on BLT, an open-source project template and tool that enables building, testing, and deploying Drupal installations following Acquia Professional Services best practices.
+This project is based on BLT, an open-source project template and tool that enables building, testing, and deploying Drupal installations following Acquia Professional Services best practices. While this is one of many methodologies, it is our recommended methodology. 
+
+* Review the [Required / Recommended Skills](http://blt.readthedocs.io/en/latest/readme/skills) for working with a BLT project
+* Ensure that your computer meets the minimum installation requirements (and then install the required applications). See the [System Requirements](http://blt.readthedocs.io/en/latest/INSTALL/#system-requirements).
+* Request access to organization that owns the project repo in GitHub (if needed)
+* Fork the project repository in GitHub
+* Request access to the Acquia Cloud Environment for your project (if needed)
+* Setup a SSH key that can be used for GitHub and the Acquia Cloud (you CAN use the same key)
+    * [Setup GitHub SSH Keys](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/)
+    * [Setup Acquia Cloud SSH Keys](https://docs.acquia.com/acquia-cloud/ssh/generate)
+* Clone your fork
+```
+$ git clone git@github.com:<account>/<project>.git
+```
+* Add the parent repository as an upstream
```

Comment:
Wouldn't this already be done by the `git clone`?

Reply:
will make this clearer in regards to origin == fork and upstream == original repository